pages/linux/mdbook.md

```diff
@@ -0,0 +1,24 @@
+# mdbook
+
+> Create online books by writing makrdown files.
+> More information: <https://rust-lang.github.io/mdBook/>.
```

Comment:
I don't find this link useful, because the CLI args are documented here:
```suggestion
> More information: <https://github.com/rust-lang/mdBook>.
```
Should we change @owenvoke ?

Reply:
I think the current one seems fine to me. 👍🏻